client/src/components/loginForm.jsx

```diff
@@ -1,22 +1,56 @@
 import React from 'react';
 
-const LoginForm = () => {
-  return (
-    <form>
-    <div id="username-container">
-      <label htmlFor="email" id="email">
-    Email address
-        <input type="text" name="email" placeholder="Enter username..." />
-      </label>
-    </div>
-    <div id="password-container">
-      <label htmlFor="password" id="password">
-    Password
-        <input type="password" name="password" />
-      </label>
-    </div>
-  </form>
-  )
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: '',
+      strength: '',
+      color: ''
+    };
+  }
+
+  handleInput = e => {
+    this.setState({
+      [`${e.target.name}`]: e.target.value
+    });
+  };
+
+  render() {
+    return (
+      <form>
+        <div id='username-container'>
+          <label htmlFor='username' id='username'>
+            <span>Username</span>
+            <input
+              onChange={this.handleInput}
```

Comment:
I might be wrong, but doesn't onChange this.handleInput need to be bound?

Reply:
Using the @babel/plugin-proposal-class-properties plugin allows for ES6 arrow functions to be used as methods on a class. 

The class properties plugin will auto bind this to the current class, so there isn't a need for binding within the constructor.